LayoutTests/inspector/debugger/async-stack-trace-basic-expected.txt

```diff
@@ -2,6 +2,26 @@ Tests for checking that async stack traces exist when pausing in asynchronous ca
 
 
 == Running test suite: AsyncStackTrace.Basic
+-- Running test case: AsyncStackTrace.Basic.Microtask
+PAUSED
+CALL STACK:
+0: [F] handleMicrotask
+ASYNC CALL STACK:
+1: --- queueMicrotask ---
+2: [F] triggerMicrotask
+3: [P] Global Code
+
+-- Running test case: AsyncStackTrace.Basic.NestedMicrotask
```

Comment:
Is there a fun test with a long stack of all the variants (promise, microtask, setTimeout, setInterval, something I'm not thinking of)?